src/fiatExchanges/SelectProvider.tsx

```diff
@@ -0,0 +1,516 @@
+  const [expanded, setExpanded] = useState(false)
+
+  const toggleExpanded = () => {
+    LayoutAnimation.easeInEaseOut()
+    setExpanded(!expanded)
+  }
+  if (!sectionQuotes.length) {
+    setNoPaymentMethods(true)
+    return null
+  }
+  const renderFeeAmount = (quote: SimplexQuote | ProviderQuote, postFix: string) => {
+    const feeAmount = getFeeValueFromQuotes(quote)
+
+    if (feeAmount === undefined) {
+      return null
+    }
+
+    return (
+      <Text>
+        <CurrencyDisplay
+          amount={{
+            value: 0,
+            localAmount: {
+              value: feeAmount,
+              currencyCode: localCurrency,
+              exchangeRate: 1,
+            },
+            currencyCode: localCurrency,
+          }}
+          showLocalAmount={true}
+          hideSign={true}
+          style={styles.fee}
+        />{' '}
+        {postFix}
+      </Text>
+    )
+  }
+  return (
+    <View style={styles.container}>
+      <Touchable
+        onPress={
+          isExpandable
+            ? toggleExpanded
+            : ((quoteOnPress(sectionQuotes[0]) as unknown) as () => void)
+        }
+      >
+        <View>
+          <Expandable
+            arrowColor={colors.greenUI}
+            containerStyle={{
+              ...styles.expandableContainer,
+              paddingVertical: isExpandable ? (expanded ? 22 : 27) : 16,
+            }}
+            isExpandable={isExpandable}
+            isExpanded={expanded}
+          >
+            {isExpandable ? (
+              <>
+                <View style={styles.left}>
+                  <Text style={styles.category}>
+                    {paymentMethod === PaymentMethod.Card
+                      ? t('selectProviderScreen.card')
+                      : t('selectProviderScreen.bank')}
+                  </Text>
+                  {!expanded && (
+                    <Text style={styles.fee}>
+                      {renderFeeAmount(sectionQuotes[0].quote, t('selectProviderScreen.minFee'))}
+                    </Text>
+                  )}
+                </View>
+
+                <View style={styles.right}>
+                  <Text style={styles.providerDropdown}>
+                    {t('selectProviderScreen.numProviders', { count: sectionQuotes.length })}
+                  </Text>
+                </View>
+              </>
+            ) : (
+              <>
+                <View style={styles.left}>
+                  <Text style={styles.category}>
+                    {paymentMethod === PaymentMethod.Card
+                      ? t('selectProviderScreen.card')
+                      : t('selectProviderScreen.bank')}
+                  </Text>
+                  <Text style={styles.fee}>
+                    {renderFeeAmount(sectionQuotes[0].quote, t('selectProviderScreen.fee'))}
+                  </Text>
+                  <Text style={styles.topInfo}>
+                    {t('selectProviderScreen.idRequired')} |{' '}
```

Comment:
Does Ramp require ID now?

Reply:
might be a good idea to conditionally show `ID Required` text, I'm worried this will hurt cash in rate

Reply:
I ran it by Laura when I was working on the flow and she said its okay. The problem is our existing external providers don't provide that info so we'd have to store it somewhere and we are trying to avoid having more detailed data that can easily go stale like that per partner.